components/gc-most-requested/index.json-ld

```diff
@@ -168,9 +209,63 @@
 					}
 				]
 			}
+		},
+		{
+			"@id": "_:implement_mostrequested_highlight",
+			"iteration": "_:iteration_mostrequested_highlight_1",
+			"name": {
+				"en": "Standard",
+				"fr": "Standard"
+			},
+			"introduction": {
+				"en": "This implementation is meant for developers/publishers adding the component manually.",
+				"fr": "Cette implémentation est destinée aux développeurs/éditeurs qui ajoutent le composant manuellement."
```

Comment:
Add a mention that they must look at the guidance to know when or not to use it.


```suggestion
				"en": "This implementation is meant for developers/publishers adding the component manually. Please check with the guidance on when you can use such pattern.",
				"fr": "Cette implémentation est destinée aux développeurs/éditeurs qui ajoutent le composant manuellement. Veuillez verifié le context d'utilisation de ce pattern selon les lignes directrices"
```